snooty.toml

```
@@ -4,5 +4,14 @@ title = "Visual Studio Extension"
 [constants]
 version = 1.0
 full-version = "{+version+}.0"
+
 name = "MongoDB Visual Studio Extension"
```

comment: `product` might be better than name 
```suggestion
product = "MongoDB Visual Studio Extension"
```